test/GeneralRateModel2D.cpp

```diff
@@ -23,42 +29,43 @@
 #include "ModelBuilderImpl.hpp"
 #include "cadet/FactoryFuncs.hpp"
 #include "ParallelSupport.hpp"
+#include "Approx.hpp"
 
-TEST_CASE("GRM2D LWE forward vs backward flow", "[GRM2D],[Simulation]")
+TEST_CASE("GRM2D LWE forward vs backward flow", "[GRM2D],[Simulation],[fixGRM2D]") // todo fix. off by a lot
 {
 	// Test all WENO orders
 	for (unsigned int i = 1; i <= cadet::Weno::maxOrder(); ++i)
 		cadet::test::column::testWenoForwardBackward("GENERAL_RATE_MODEL_2D", i, 1e-9, 2e-4);
 }
 
-TEST_CASE("GRM2D Jacobian forward vs backward flow", "[GRM2D],[UnitOp],[Residual],[Jacobian]")
+TEST_CASE("GRM2D Jacobian forward vs backward flow", "[GRM2D],[UnitOp],[Residual],[Jacobian],[fixGRM2D]") // todo fix. off by some tolerance
 {
 	// Test all WENO orders
 	for (unsigned int i = 1; i <= cadet::Weno::maxOrder(); ++i)
 		cadet::test::column::testJacobianWenoForwardBackwardFD("GENERAL_RATE_MODEL_2D", i, 1e-6, 0.0, 1e-3);
 }
 
-TEST_CASE("GRM2D time derivative Jacobian vs FD", "[GRM2D],[UnitOp],[Residual],[Jacobian]")
+TEST_CASE("GRM2D time derivative Jacobian vs FD", "[GRM2D],[UnitOp],[Residual],[Jacobian],[FDtestGRM2D]")
 {
 	cadet::test::column::testTimeDerivativeJacobianFD("GENERAL_RATE_MODEL_2D", 1e-6, 0.0, 9e-4);
 }
 
-TEST_CASE("GRM2D rapid-equilibrium binding flux Jacobian vs FD", "[GRM2D],[UnitOp],[Residual],[Jacobian]")
+TEST_CASE("GRM2D rapid-equilibrium binding flux Jacobian vs FD", "[GRM2D],[UnitOp],[Residual],[Jacobian],[FDtestGRM2D]")
 {
-	cadet::test::column::testArrowHeadJacobianFD("GENERAL_RATE_MODEL_2D", false, 1e-6, 2e-9);
```

Comment:
Is there some reasoning behind these values or is it just "trial and error" until it runs? Might be a good idea to explain this somewhere, but not strictly necessary.

Reply:
iam planning on adding a description of this in the [developer guide](https://github.com/modsim/CADET/blob/chore/developer_guide/doc/developer_guide/testing.rst)
Ive added this to the todo list so that we dont forget about it